terraform/mash-setup-staging/main.tf

```diff
@@ -0,0 +1,21 @@
+provider "aws" {
+  region  = "eu-west-2"
+  version = "~> 2.0"
```

Comment:
Version 2 here clashes with version three given below. You can just remove this as you have it configured under required_providers below.

Reply:
Thanks, I'll remove the version.
I think I'll still need the provider block so that I can specify a region. It failed for me the first time because I hadn't specified a region and the required_provider block doesn't accept region as an input.

Reply:
Done in a14accf